### DISCUSSION TITLE
Automatic Diary Task and Alert

### DISCUSSION THREAD

**Discussion by angel4bh** [sheltermanager/asm3#1555] [Q&A / Help]
I want to set up a diary task that will create an alert to follow up on an adoption after a set period of time (like 30 days after) to make sure that everything is going well with the placement. Ideally, the diary task would be automatically set once the movement to adoption is saved. If possible, I'd like to avoid having the coordinator responsible for manually entering this task for every adopted animal as humans are prone to forget things. 

On a side note...is there a sandbox mode to shelter manager?

**Comment by bobintetley** [comment]
There's no way to automatically create diary tasks, but there is an option to automatically email adopters after a fixed period of time. Perhaps this might do what you're looking for? The option is under Settings->Options->Reminder Emails.

![image](https://github.com/user-attachments/assets/ca78a51e-4053-41f4-893a-a5099206e26e)
